x-pack/plugins/cloud_security_posture/public/common/navigation/constants.ts

```diff
@@ -27,7 +27,7 @@ const NAV_ITEMS_NAMES = {
     defaultMessage: 'Findings',
   }),
   BENCHMARKS: i18n.translate('xpack.csp.navigation.myBenchmarksNavItemLabel', {
-    defaultMessage: 'Cloud Posture Benchmarks',
+    defaultMessage: 'Benchmark rules',
```

Comment:
if the wording change has been approved by design/product, from Cloud Security Posture looks good. It's just looks a bit confusing when presented in the navigation without any mention of Cloud Security